### PR TITLE
Objects Automated Test - LPS-138961 | master

### PR DESCRIPTION
**Pull request by PauloVAS**

### CODE REVIEW

**Comment by PauloVAS** [comment]
ci:test:sf

File: modules/apps/object/object-test/src/testFunctional/tests/macros/ObjectPortlet.macro [hunk 1]
```diff
@@ -6,6 +6,30 @@ definition {
...
+		AssertElementPresent(
+			key_rowNumber = "${rowNumber}",
+			locator1 = "ObjectPortlet#ENTRY_ROW_NUMBER");
+	}
+
+	macro assertEntryRowNotDisplayed {
+		AssertElementNotPresent(
+			key_rowNumber = "${rowNumber}",
+			locator1 = "ObjectPortlet#ENTRY_ROW_NUMBER");
+	}
```
Comment (rodrigocunhaa) [inline modules/apps/object/object-test/src/testFunctional/tests/macros/ObjectPortlet.macro:19]: You can remove this.

File: modules/apps/object/object-test/src/testFunctional/tests/paths/ObjectPortlet.path [hunk 2]
```diff
@@ -10,11 +10,21 @@
...
 	<tr>
 		<td>ENTRY_VALUE</td>
 		<td>//div[contains(@class,'dnd-tbody')]//div[contains(text(),'${key_entry}')]</td>
 		<td></td>
 	</tr>
+	<tr>
+		<td>ENTRY_ROW_NUMBER</td>
+		<td>//div[contains(@class,'dnd-tr')][${key_rowNumber}]</td>
+		<td></td>
+	</tr>
```
Comment (rodrigocunhaa) [inline modules/apps/object/object-test/src/testFunctional/tests/paths/ObjectPortlet.path:27]: you can remove this

File: modules/apps/object/object-test/src/testFunctional/tests/tests/ObjectPortlet.testcase [hunk 3]
```diff
@@ -201,12 +201,44 @@ definition {
...
+			fieldName = "customObjectField",
+			fieldType = "String",
+			isRequired = "false",
+			objectName = "CustomObject");
+
+		ObjectAdmin.publishObjectViaAPI(objectName = "CustomObject");
+
+		for (var number : list "1,2,3,4,5") {
+			ObjectAdmin.addObjectSingleFieldEntryViaAPI(
+				fieldName = "text",
```
Comment (rodrigocunhaa) [inline modules/apps/object/object-test/src/testFunctional/tests/tests/ObjectPortlet.testcase:222]: Change this to:
`fieldName = "customObjectField",`

File: modules/apps/object/object-test/src/testFunctional/tests/tests/ObjectPortlet.testcase [hunk 4]
```diff
@@ -201,12 +201,44 @@ definition {
...
+				objectName = "CustomObject",
+				value = "Test text");
```
Comment (rodrigocunhaa) [inline modules/apps/object/object-test/src/testFunctional/tests/tests/ObjectPortlet.testcase:224]: Change this to:
value = "Test text ${number}"

File: modules/apps/object/object-test/src/testFunctional/tests/tests/ObjectPortlet.testcase [hunk 5]
```diff
@@ -201,12 +201,44 @@ definition {
...
+		ObjectPortlet.assertEntryRowIsPresent(rowNumber = "1");
+
+		ObjectPortlet.assertEntryRowNotDisplayed(rowNumber = "2");
+
+		ObjectPortlet.assertEntryRowNotDisplayed(rowNumber = "3");
 
-		// TODO LPS-138961 TableUpdatedWhenChangingPage pending implementation
+		ObjectPortlet.assertEntryRowNotDisplayed(rowNumber = "4");
 
+		ObjectPortlet.assertEntryRowNotDisplayed(rowNumber = "5");
```
Comment (rodrigocunhaa) [inline modules/apps/object/object-test/src/testFunctional/tests/tests/ObjectPortlet.testcase:241]: Use `ObjectPortlet.viewEntry` and `ObjectPortlet.assertEntryNotPresent` here.

Check this as reference: https://github.com/rodrigocunhaa/liferay-portal/pull/1038/files

**Review by rodrigocunhaa — CHANGES_REQUESTED** [review]
Hi @PauloVAS ,

Please, check the changes that I requested.

Thanks!

**Comment by PauloVAS** [comment]
@rodrigocunhaa, fixed!

**Comment by PauloVAS** [comment]
ci:test:sf

**Comment by PauloVAS** [comment]
ci:test:sf

**Comment by rodrigocunhaa** [comment]
ci:forward

**Comment by rodrigocunhaa** [comment]
Just started reviewing :)

**Pull request closed without merging**